src/strings/PalindromOrNot.java

```diff
@@ -0,0 +1,30 @@
+package strings;
+
+import java.util.Scanner;
+
+public class PalindromOrNot {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String str = scanner.nextLine();
+
+        if (isPalindrome(str))
+            System.out.print("Is Palindrome");
+        else
+            System.out.print("Is Not Palindrome");
+
+    }
+
+    static boolean isPalindrome(String str)
```

Comment:
if (yourString.equals(new StringBuilder(yourString).reverse())) --> yes, it's polidrome!

:) start to use string builder